amundsen_application/static/js/components/TableDetail/ColumnListItem/ColumnType/styles.scss

```diff
@@ -11,6 +11,18 @@ $modal-header-height: 94px;
 // TODO (design qa): Also using fixed width from Figma
 $modal-dialog-width: 418px;
 
+button.column-type {
```

Comment:
I am mostly against styling using tags, as it makes harder to solve specificity issues.
I would recommend instead to create a new class, something like `.column-type-btn` and style that one.